### DISCUSSION TITLE
Elasticsearch index export

### DISCUSSION THREAD

**Discussion by yula-21** [Security-Onion-Solutions/securityonion#8540] [Unsupported Versions] [answered]
Hi,
Is there any way to export elasticsearch indexes from Security Onion to a different elastic server?
I know theres a way to export indexes as csv but i need to export the index and then import it again.

**Comment by dougburks** [comment, marked as answer]
We don't provide any support for this, but you might try something like https://github.com/elasticsearch-dump/elasticsearch-dump.

**Comment by weslambert** [comment]
You can also reindex to a remote cluster, or create/restore a snapshot:

https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-reindex.html#reindex-from-remote
https://www.elastic.co/guide/en/elasticsearch/reference/current/snapshot-restore.html
